### PR TITLE
Add a DeFi guide

### PR DESCRIPTION
**Pull request by JanetMo**
Adding a guide about DeFi, #32

Feeld free to adapt it or let met know what to change. I tried to include some developer resources but there's probably much more (-:

### CODE REVIEW

File: content/guides/intro-to-defi.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+- **Pseudonymous**: People act through their wallet addresses, no personal information needed.
+- **Transparent**: Transactions are public, though, and often the code of the applications is open-source.
+
+DeFi essentially extends the "digital money" premise of bitcoin beyond simply making payments with digital currency. DeFi applications allow you to trade crypto, create derivatives, borrow and lend, earn interest or buy insurance. In the next section, we will explain the broader areas of DeFi.
+
+Some intro pages and learning resources to start your DeFi journey:
+
+- [_Ethereum.org DeFi intro page_](https://ethereum.org/en/defi)
+- [_Rekt news_](https://rekt.news/de/)
+- [_Finematics Youtube channel_](https://www.useweb3.xyz/videos/finematics)
```
Comment (wslyvh) [inline content/guides/intro-to-defi.md:37]: I'd link to the finematics channel directly

File: content/guides/intro-to-defi.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+- [_The Defiant news_](https://thedefiant.io/)
+- [_How to DeFi Advanced Book_](https://www.useweb3.xyz/books/how-to-defi-advanced)
```
Comment (wslyvh) [inline content/guides/intro-to-defi.md:39]: I wonder how up to date this still is?

File: content/guides/intro-to-defi.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+- [_Bankless Podcast_](https://www.useweb3.xyz/podcasts/bankless)
```
Comment (wslyvh) [inline content/guides/intro-to-defi.md:40]: Link to Bankless directly

File: content/guides/intro-to-defi.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+- [_DeFi course by the University of Nicosia_](https://www.useweb3.xyz/courses/defi-mooc)
```
Comment (wslyvh) [inline content/guides/intro-to-defi.md:41]: Same as above. Not sure how up to date it still is

File: content/guides/intro-to-defi.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+One of the biggest risks in DeFi is the loss of funds due to hacks and other security issues. In recent years, there have been hacks that have drained up to $600 million from DeFi applications. Therefore, becoming a DeFi security specialist can be a promising career path.
+
+Security breaches, such as hacks or exploits, can result in significant financial losses for users and damage to the platform's reputation. By focusing on DeFi security, you can help mitigate these risks and protect users' assets.
+
+Learn more about DeFi hacks and how to prevent them:
+
+- [_Rekt leaderboard_](https://rekt.news/leaderboard/)
+- [_DeFi Exploit timeline_](https://chainsec.io/defi-hacks/)
+- [_GitHub DeFi risk tools_](https://github.com/defi-defense-dao/defi-risk-tools-list)
+- [_Damn Vulnerable DeFi code challenges_](https://www.useweb3.xyz/code-challenges/damn-vulnerable-defi)
```
Comment (wslyvh) [inline content/guides/intro-to-defi.md:90]: Link to Damn vulnerable DeFi directly

**Comment by wslyvh** [comment]
This is great! Thanks for writing another guide @JanetMo :100: 

Left a few comments on some of the resources. I'm not too familiar in DeFi myself that I'm not sure what resources are still up to date and relevant. But otherwise, this seems great. Happy to merge :)

File: content/guides/intro-to-defi.md (hunk 2 above)
Comment (JanetMo) [reply content/guides/intro-to-defi.md:39]: reight, from 2021, probably not the most up to date...

File: content/guides/intro-to-defi.md (hunk 4 above)
Comment (JanetMo) [reply content/guides/intro-to-defi.md:41]: ah thanks, I put the wrong course, the one from Nicosia is every semester

**Comment by JanetMo** [comment]
You're very welcome, thanks for checking 👍
I adapted the indirect links and removed the outdated ones. (-:

**Pull request merged**